File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/VentaDetalleResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class VentaDetalleResponse {
    private Long id;
    private Long loteDetalleId;
    private String medicamentoNombre;
    private String presentacion;
    private String loteNumero;
    private Integer cantidad;
    private BigDecimal precioUnitario;
    private BigDecimal subtotal;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/VentaDetalleResponse.kt
package com.sanidad.movil.data.remote.dto

data class VentaDetalleResponse(
    val id: Long,
    val loteDetalleId: Long,
    val medicamentoNombre: String,
    val presentacion: String,
    val loteNumero: String,
    val cantidad: Int,
    val precioUnitario: Double,
    val subtotal: Double
)